nuntium/popit_api_instance.py

```diff
@@ -1,9 +1,34 @@
 from popit.models import ApiInstance, Person, get_paginated_generator
 from contactos.models import Contact
 from mailit import MailChannel
+from datetime import datetime
 
 
-class PopitPerson(Person):
+class IsActiveMixin(object):
+    format = "%Y-%m-%d"
+
+    @classmethod
+    def get_date_or_none(cls, doc, key):
+        try:
+            return datetime.strptime(doc.get(key, None), cls.format)
+        except TypeError:
+            return None
+
+    @classmethod
+    def _is_active(cls, start_date, end_date):
+        today = datetime.today()
+        is_valid = ((start_date is None) or start_date <= today) and\
+            ((end_date is None) or end_date >= today)
+        return is_valid
+
+    @classmethod
+    def is_active(cls, doc, start_date_key='start_date', end_date_key='end_date'):
```

Comment:
It took me a while to realise that `doc` was going to be a membership, from a json file. Perhaps give it a different name to distinguish it from the `doc` in the method below (how about `membership`?)